### PR TITLE
Support extra parameters for the ssm-session command.

### PR DESCRIPTION
**Pull request by arthurio**
Based on:
  - https://github.com/mludvig/aws-utils/pull/2 from @aws-taylor.
  - https://github.com/aws/amazon-ssm-agent/issues/131#issuecomment-622957074 from @AndreCouture

Example:

```shell
ssm-session my-instance.hostname --document-name AWS-StartInteractiveCommand --parameters command='["sudo -i -u myuser"]'
```

### CODE REVIEW

File: ssm_tools/ssm_session_cli.py [hunk 1]
```diff
@@ -57,21 +57,26 @@ def parse_args(argv):
...
         extra_args += f"--profile {profile} "
     if region:
         extra_args += f"--region {region} "
-    command = f'aws {extra_args} ssm start-session --target {instance_id}'
+
+    parameters = ""
+    for extra in extras:
+        parameters += f" {extra}"
+
+    command = f'aws {extra_args} ssm start-session --target {instance_id}{parameters}'
```
Comment (mludvig) [inline ssm_tools/ssm_session_cli.py:73]: Should there be a space in `{instance_id}{parameters}`?

Comment (arthurio) [reply ssm_tools/ssm_session_cli.py:73]: No because we prefix a space in the for loop. That way if there are no parameters, we won’t add an extra space.

Comment (mludvig) [reply ssm_tools/ssm_session_cli.py:73]: True. So maybe `parameters = " ".join(extras)` in that case?

Comment (mludvig) [reply ssm_tools/ssm_session_cli.py:73]: And by the way an extra space wouldn't really matter while making the string more _aesthetically pleasing_ :)

Comment (arthurio) [reply ssm_tools/ssm_session_cli.py:73]: Sounds good to me 👍

**Comment by mludvig** [comment]
Merged, thanks for the contribution!

**Comment by arthurio** [comment]
You are welcome, thanks for all the great tools!

**Comment by mludvig** [comment]
FYI just added `--user USER` as a shortcut for specifying the ssm documents and parameters. I suppose setting the username will be the most common usecase anyway.

**Comment by AndreCouture** [comment]
Thanks guys

Can’t wait to try it !


—
Regards,

Andre Couture
Senior Cloud Solution Architect 

andre.couture@me.com
https://www.linkedin.com/in/andrecouture

**Pull request merged**